sample.py

```diff
@@ -1,2 +1,3 @@
 import os
```

Comment:
This is a Unusedimportos
```suggestion 
```

 from our GitHub Enhacement Bot